platforms/documentation/docs/src/docs/userguide/reference/glossary.adoc

```diff
@@ -15,6 +15,25 @@ Directories are common in the case of inter-project dependencies to avoid the co
 
 == B
 
+[[sub:terminology_build]]
+Build::
```

Comment:
💅 We should give some credits to https://github.com/autonomousapps/gradle-glossary, maybe in a form of a mention in the list of contributors in the release notes.

Reply:
Added.